assets/js/base/context/providers/cart-checkout/payment-methods/actions.ts

```diff
@@ -14,72 +14,68 @@ import { ACTION, STATUS } from './constants';
 export interface ActionType {
 	type: ACTION | STATUS;
 	errorMessage?: string;
-	paymentMethodData?: Record< string, unknown >;
+	paymentMethodData?: Record< string, unknown > | undefined;
 	paymentMethods?: PaymentMethods | ExpressPaymentMethods;
+	paymentMethod?: string;
 	shouldSavePaymentMethod?: boolean;
 }
 
 /**
  * All the actions that can be dispatched for payment methods.
  */
 export const actions = {
-	statusOnly: ( type: STATUS ): { type: STATUS } => ( { type } as const ),
-	error: ( errorMessage: string ): ActionType =>
-		( {
-			type: STATUS.ERROR,
-			errorMessage,
-		} as const ),
+	statusOnly: ( type: STATUS ): ActionType => ( {
+		type,
+	} ),
+	error: ( errorMessage: string ): ActionType => ( {
+		type: STATUS.ERROR,
+		errorMessage,
+	} ),
 	failed: ( {
 		errorMessage,
 		paymentMethodData,
 	}: {
 		errorMessage: string;
 		paymentMethodData: Record< string, unknown >;
-	} ): ActionType =>
-		( {
-			type: STATUS.FAILED,
-			errorMessage,
-			paymentMethodData,
-		} as const ),
+	} ): ActionType => ( {
+		type: STATUS.FAILED,
+		errorMessage,
+		paymentMethodData,
+	} ),
 	success: ( {
 		paymentMethodData,
 	}: {
 		paymentMethodData?: Record< string, unknown >;
-	} ): ActionType =>
-		( {
-			type: STATUS.SUCCESS,
-			paymentMethodData,
-		} as const ),
-	started: ( {
```

Comment:
So we can no longer dispatch `started`?

Reply:
No, it's dispatched as `statusOnly` now like pristine. That's all express methods need. The data addition was added for saved payment methods - with the new `setActivePaymentMethod` method that's no longer a requirement. They can stay pristine.